Dockerfile

```diff
@@ -20,29 +42,55 @@ RUN apk update && \
         proj-util \
         proj-dev \
         make
+# Setup backend Python dependencies
+COPY --from=extract-deps \
+    /opt/python/requirements.txt /opt/python/
+RUN pip install --user --no-warn-script-location \
```

Comment:
I'm assuming `--no-warn-script-location` because we are installing from `/opt/python/requirements.txt` which is not in PYPATH

Reply:
I decided on that flag so long ago now I have forgotten why 😄 Presumably it's to suppress the warning, but I can't seem to reproduce a warning on the `python:3.9` image, so perhaps it's been updated since.